core/php7.1Action/CHANGELOG.md

```diff
@@ -19,8 +19,9 @@
 
 ## Apache 1.13.0-incubating (next release)
 Changes:
-  - Update version of PHP to 7.1.26
+  - Update version of PHP to 7.1.27
 
+## 1.12.0-incubating
```

Comment:
Curious about `1.12.0-incubating`. Are there no changes in that release?

Reply:
I added the tag since I have seen this in the docker registry (e.g. https://hub.docker.com/r/openwhisk/action-php-v7.1/tags) and it was not documented in the CHANGELOG.md of the individual runtime images, yet.

I looked at the history.
The change from 1.0.3 to 1.12.0-incubating was to revert the content of copyright to the content in the template in LICENSE.txt (https://github.com/apache/incubator-openwhisk-runtime-php/commit/93c1871421fc92441cab8d4b1d23a31f18d7506e) on Aug 28, 2018. This change is not related to the generated docker images.
Means, the images of 1.0.3 and 1.12.0-incubating are functional identical. Which I tried to visualize by not having a 'Changes' section in between the 2 versions.

In case there is a more common way to show this, I can adjust it accordingly :-).